cmd/apiserver/main.go

```diff
@@ -36,12 +38,16 @@ func main() {
 	flag.StringVar(&s.restCfg.Datastore.Type, "datastore-type", "kubeapi", "Metadata storage driver type, support kubeapi and mongodb")
 	flag.StringVar(&s.restCfg.Datastore.Database, "datastore-database", "kubevela", "Metadata storage database name, takes effect when the storage driver is mongodb.")
 	flag.StringVar(&s.restCfg.Datastore.URL, "datastore-url", "", "Metadata storage database url,takes effect when the storage driver is mongodb.")
+	flag.StringVar(&s.restCfg.LeaderConfig.ID, "id", uuid.New().String(), "the holder identity name")
```

Comment:
Why implement leader election? And why assume it is on k8s?

Reply:
We need a goroutine to sync the status of cluster and database, the goroutine need leader election to ensure there is only one goroutine is working in multiple replicas.

Our APIServer is working on a manage cluster so that it can deploy application.